algorithms/corpus_creators/basic.py

```diff
@@ -30,6 +30,21 @@ def raw_corpora(self, generator):
             document = json.loads(line)
             yield self._transform(document)
 
+    def array_corpora(self, generator):
```

Comment:
Docstring would be helpful here, especially since *how* it produces the array (splitting on empty string) is pretty important to know.